### DISCUSSION TITLE
MIT License

### DISCUSSION THREAD

**Discussion by CSharpDummy** [FreeOpcUa/opcua-asyncio#532] [General]
Any chance you could change the license to MIT. Industrial apps are hesitant to use any other license.

**Comment by AndreasHeine** [comment]
as @oroulet mentioned: No

> And I answer: No
> Would need to get everyone that has contributed to agree. It is never going to happen.
> and also LGPL should be good engough for everything you want apart from copying the code inside your applicaiton

**Reply by AndreasHeine** [reply]
actually if you compare both you will notice that LGPL is more open while mantaining the protection of your code

**Reply by AndreasHeine** [reply]
https://github.com/FreeOpcUa/opcua-asyncio/blob/master/COPYING

https://github.com/git/git-scm.com/blob/main/MIT-LICENSE.txt
